internal/external-dns/plan/plan.go

```diff
@@ -417,6 +417,14 @@ type endpointUpdate struct {
 }
 
 func (e *endpointUpdate) ShouldUpdate() bool {
+	desiredTarget := e.desired.Targets[0]
+	currentTarget := e.current.Targets[0]
+	_ = desiredTarget + currentTarget
```

Comment:
doesn't look to be used?

Reply:
Ah, missed those. Leftowers from running controller in debug mode.